OxfordBodleian/Juel-Jensen/BDLaethf31.xml

```diff
@@ -312,42 +312,45 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                 </item>
 
                                 <item xml:id="e1">
-                                    <desc>A white sticker is glued to the inner side of the front board. The name of <persName ref="PRS5782JuelJen"/> is printed on it
-                                        (<foreign xml:lang="gez">ቤንት፡ ዩውል፡ የንሰን።</foreign>) and the signature "MS. Aeth. f. 31" is written in pencil.
-                                        An additional sticker, with the signature "Ms 21" on it, is glued below it.</desc>
-                                   <!-- quotation mark-->
-                                </item>
-                                <item xml:id="e2">
-                                    <desc>There are some writings in pencil on <locus target="#1r"/>: <q xml:lang="en">"Brought from
-                                        <persName ref="PRS13027Sturt" role="owner"><roleName type="rank">Capt.</roleName> Sturt</persName> from
-                                        <placeName ref="LOC3010Ethiop">Ethiopia</placeName>"</q> and
-                                        <q xml:lang="en">"Gospel of John"</q>. The titles of the manuscript contents are indicated in the same handwriting in pencil in the upper margin of
-                                        <locus target="#3r"/> (<q xml:lang="en">"Gospel of John"</q>) and <locus target="#75v"/> (<q xml:lang="en">"Hymns to the Virgin"</q>).
+                                    <desc>Many corrections are executed over erasure (e.g. on <locus target="#10ra #20va #38va"/>), sometimes in small script to accommodate multiple lines (e.g.
+                                        <locus target="#23rb #31rb #40va"/>). Many corrections, executed either over erasure or interlineally, are written in the same hand as that of <ref target="#e3">the
+                                            aids for identifying weekday readings</ref>. Erasures used for corrections are sometimes marked with thin lines (e.g. on <locus target="#11rb"/>).
+                                        Erasures are also found in the upper margins in correspondence of the incipit of the weekday readings (e.g. on <locus target="#16vb #40vb #69rb"/>).
+                                        The name of an individual, indicated as <foreign xml:lang="gez">ገብሩ፡</foreign> ‘his servant’ (probably either the owner, the patron or the donor of
```

Comment:
It could also be the name "Gabru"?

Reply:
@MassimoVilla I think ገብሩ introduces the name, which has been erased